docs/whats-new.md

```diff
@@ -4,30 +4,49 @@ This page contains highlights of each deck.gl release. Also check our [vis.gl bl
 
 ## deck.gl v9.2
 
-Target release date: July, 2025
+Target release date: September, 2025
+
+### Widgets
+
+- A suite of new widgets have been added to the [`@deck.gl/widgets`](./api-reference/widgets/overview.md) module:
+  - ContextMenuWidget
```

Comment:
- Some of these are less mature / more experimental and it was kind of semi-intentional that they were omitted here.
- It might be worth going through widgets/index.ts and review what should be exported with underscore and what should not.

Reply:
They are marked as experimental in the docs:
<img width="282" height="127" alt="Screenshot 2025-09-24 at 08 52 25" src="https://github.com/user-attachments/assets/0781318b-6ff6-462d-b227-80fad4582488" />

But, I don't feel that strongly here, and am not clued up on which widgets are deemed more stable, so will leave it to you to remove any underscores if you like

I think it would be a shame to move them to `@deck.gl-community/widgets`

Reply:
I'm going to make some direct updates to this branch for consistency if that's alright

Reply:
I pushed some up but then found a lot more changes I want to make so I'm going to open a new PR